src/gui/sharee.h

```diff
@@ -67,7 +67,7 @@ class ShareeModel : public QAbstractListModel
     explicit ShareeModel(const AccountPtr &account, const QString &type, QObject *parent = nullptr);
 
     using ShareeSet = QVector<QSharedPointer<Sharee>>; // FIXME: make it a QSet<Sharee> when Sharee can be compared
-    void fetch(const QString &search, const ShareeSet &blacklist);
+    void fetch(const QString &search, const ShareeSet &blacklist, bool lookup);
```

Comment:
I'm not a huge fan on bools in API like that, this reduces readability at the code using it:
https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

I'd thus propose using an enum for this declared in `ShareeModel`, could be named `LookupMode` or such.

Alternative would be to have two public functions, one for each mode + a private method used by both to reduce code duplication. This has the advantage of reducing the number of parameters but at the price of more boilerplate. Maybe something to reserve only for ShareUserGroupWidget since it'd simply the connects?

Reply:
@er-vin Added